src/System.Net.HttpListener/tests/GetContextHelper.cs

```diff
@@ -59,7 +59,7 @@ public class CustomAsyncResult : IAsyncResult
     public static class Helpers
     {
         public static bool IsWindowsImplementation { get; } = PlatformDetection.IsWindows && PlatformDetection.IsNotOneCoreUAP;
-        public static bool IsNotWindowsImplementation => !IsWindowsImplementation;
+        public static bool IsNotWindowsImplementation => !IsWindowsImplementation && PlatformDetection.IsNotOneCoreUAP;
```

Comment:
>  httpapi.dll which apparently is not available on Nano. Can you confirm this change is correct?

Nano is package based - depends on how it's configured. IIS works on Nano and so I'd expect httpapi.dll to be available. (https://docs.microsoft.com/en-us/windows-server/get-started/iis-on-nano-server)